.github/workflows/network-deploy.yml

```diff
@@ -11,20 +11,10 @@ on:
         description: The namespace to deploy to, e.g. smoke
         required: true
         type: string
-      metrics_namespace:
-        description: The namespace of the metrics deployment, e.g. metrics
-        required: false
-        type: string
-        default: metrics
       values_file:
-        description: The values file to use, e.g. 1-validators.yaml
+        description: List of values files to use, e.g. 1-validators.yaml,gcloud.yaml. Note that 2nd file values will supersede the first one.
```

Comment:
The description says it's a list. Does terraform automatically parse it as a list if it's comma-separated?

Reply:
ah no, leftover change, will revert